server/src/main/java/org/elasticsearch/search/aggregations/support/ValuesSourceRegistry.java

```diff
@@ -169,6 +171,11 @@ public <T> T getAggregator(RegistryKey<T> registryKey, ValuesSourceConfig values
             @SuppressWarnings("unchecked")
             T supplier = (T) aggregatorRegistry.get(registryKey).get(valuesSourceConfig.valueSourceType());
             if (supplier == null) {
+                if ("aggregate_metric_double".equals(valuesSourceConfig.fieldType().typeName().toLowerCase(Locale.ROOT))) {
```

Comment:
Could not use the `AggregateDoubleMetricFieldMapper.CONTENT_TYPE` due to a dependency issue.